frontend/public/components/monitoring/alerting.tsx

```diff
@@ -1712,6 +1757,7 @@ type AlertsDetailsPageProps = {
   namespace: string;
   rule: Rule;
   silencesLoaded: boolean;
+  match?: any;
 };
 
 type AlertMessageProps = {
```

Comment:
The issue why your component was not getting rendered is because `exact` is added to the route and which means the url path and route path on which the component is supposed to render should be exactly same 
https://github.com/divyanshiGupta/console/blob/7b41f1bda5f31470cf229ef1aab7d7ec657c147e/frontend/public/components/monitoring/alerting.tsx#L1732
Remove `exact` from here